internal/controller/predicate.go

```diff
@@ -41,7 +41,7 @@ func isObjectOurs(scheme *runtime.Scheme, object client.Object) bool {
 		return false
 	}
 	gvk := objGVKs[0]
-	if gvk.Group == oadpv1alpha1.GroupVersion.Group && gvk.Version == oadpv1alpha1.GroupVersion.Version && gvk.Kind == oadpv1alpha1.Kind {
+	if gvk.Group == oadpv1alpha1.GroupVersion.Group && gvk.Version == oadpv1alpha1.GroupVersion.Version && gvk.Kind == "DataProtectionApplication" {
```

Comment:
is this string not available somewhere as const/var already?

Reply:
it was in `api/` folder

since it was only referenced here, I removed it